packages/bunyan/src/node/bunyan-logger-server.ts

```diff
@@ -68,7 +68,7 @@ export class BunyanLoggerServer implements ILoggerServer {
      * documentation.  */
     child(name: string): Promise<void> {
         if (name.length === 0) {
-            return Promise.reject("Can't create a logger with an empty name.");
+            return Promise.reject(Error("Can't create a logger with an empty name."));
```

Comment:
Weirdly enough, just doing `Error(message)` seems to work, but the usual way of creating an error is by doing `new Error(message)`. Maybe change to that and see if other places also forgot the `new` keyword?

Reply:
a single such usage could be find. Has been fixed

Reply:
Ah @kittaakos already saw that, take this as a reminder I guess :p